ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiCommandService.scala

```diff
@@ -154,11 +124,13 @@ private[apiserver] final class ApiCommandService private[services] (
   )(implicit
       loggingContext: LoggingContext,
       errorLogger: ContextualizedErrorLogger,
-  ): Future[Either[TrackedCompletionFailure, CompletionSuccess]] =
+  ): Future[CompletionResponse] =
     if (running) {
       val timeout = Option(Context.current().getDeadline)
```

Comment:
Not your code, but I am thinking whether it is good to use thread-local stuff here, especially in this method that returns a `Future[_]`. If this call is flatMapped over in the caller, it's nasty. What do you think?